libs/martech/attributes.js

```diff
@@ -1,26 +1,9 @@
 const RE_ALPHANUM = /[^0-9a-z]/gi;
 const RE_TRIM_UNDERSCORE = /^_+|_+$/g;
 
-export function decorateBlockAnalytics(blockEl) {
-  blockEl.setAttribute('daa-im', 'true');
-  blockEl.setAttribute('daa-lh', [...blockEl.classList].join('|'));
-}
-
-export function decorateLinkAnalytics(textEl, headings) {
-  if (headings) {
-    const headingText = [...headings].map((heading) => heading.textContent);
-    textEl.setAttribute('daa-lh', headingText.join('|').replace(/\s+/g, ' ').trim());
-  }
-  const links = textEl.querySelectorAll('a, button');
-  links.forEach((link, i) => {
-    let linkType = 'link';
-    const { classList } = link;
-    if (classList.contains('con-button') && classList.contains('blue')) { linkType = 'filled'; }
-    if (classList.contains('con-button') && classList.contains('outline')) { linkType = 'outline'; }
-    const str = `${linkType}|${link.innerText} ${i + 1}`;
-    link.setAttribute('daa-ll', str.replace(/\s+/g, ' ').trim());
-  });
-}
+// decorateBlockAnalytics & decorateLinkAnalytics are being sunset
+export function decorateBlockAnalytics() { return false; }
+export function decorateLinkAnalytics() { return false; }
```

Comment:
Looks like `analyticsDecorateList` is only used in footer.js.  I think it would be safe to move there and remove this file (after checking to make sure no consumers are using it)

Reply:
I see it in footer.js and gnav.js.  But this file is now really just for gnav and footer.  None of the other blocks will be using this file. so since they have it in 2 places, I figured I'd leave it in this file.

Reply:
Just as an FYI, those files are the previous versions of the navigations, currently the blocks are `global-navigation.js` and `global-footer.js`. So `gnav.js` and `footer.js` will be deprecated quite soon

Reply:
Yeah, so since I've moved all of the functions we're using in utils to the analytics.js file, the functions living there until after gnav and footer are deprecated seems fine to me.